cfbs.json

```diff
@@ -901,6 +901,30 @@
         }
       ]
     },
+    "packages-allowlist-snapshot": {
+      "description": "Allows you to snapshot which packages are installed on a system, and then enforce that list, uninstalling or giving warnings when other packages appear.",
+      "tags": ["management", "inventory", "security", "experimental"],
+      "repo": "https://github.com/olehermanse/cfengine-packages-allowlist-snapshot",
+      "by": "https://github.com/olehermanse",
+      "version": "0.0.1",
+      "commit": "9de0d75772c29452019e04e5d5da2e9799eb9fc5",
+      "steps": [
+        "copy main.cf services/cfbs/modules/packages-allowlist-snapshot/main.cf",
+        "bundles packages_allowlist_snapshot:entry_point",
+        "policy_files services/cfbs/modules/packages-allowlist-snapshot/main.cf",
+        "input packages-allowlist-snapshot/input.json def.json"
+      ],
+      "input": [
+        {
+          "type": "string",
+          "namespace": "packages_allowlist_snapshot",
+          "bundle": "state",
+          "variable": "mode",
+          "label": "Mode (init|warn|enforce)",
+          "question": "What mode should the module be in?"
```

Comment:
```suggestion
          "question": "What mode should the module be in? (init|warn|enforce)"
```

Reply:
It looks very bad to repeat this multiple times in the GUI, but I can move it to the question instead of label. [We need dropdowns.](https://northerntech.atlassian.net/browse/ENT-9678)